pkg/config/parse.go

```diff
@@ -99,3 +106,33 @@ func GetConf(path string) (*Config, error) {
 	}
 	return conf, nil
 }
+
+// checkFilePerms given a list of files
+func checkFilePerms(files ...string) error {
+	const op = "config.checkFilePerms"
+
+	for _, f := range files {
+		if f == "" {
+			continue
+		}
+
+		// TODO: Remove opening a file
+		// There is a subtle bug in the filter module which ignores the filter file if it does not find it.
+		// This check can be removed once that has been fixed
+		_, err := os.Open(f)
```

Comment:
Lstat should fail if file does not exist, isn't it acceptable enough? 
Also if this needs to stay, make sure you close it, handles will be leaking here and can cause Open on other places to fail

Reply:
Oh yes, that should be acceptable enough because Lstat only returns path errors. Let me fix that.